### ISSUE TITLE
InstantTranslate error with NVDA 2019.2.1

### ISSUE THREAD

**Issue by CyrilleB79** [addonFactory/instantTranslate#9]
## Issue description

When running NVDA 2019.2.1 with Instant Translate, I get the following error in the log:

```
ERROR - eventHandler.executeEvent (10:21:14.907):
error executing event: selection on <NVDAObjects.IAccessible.sysListView32.ListItem object at 0x054668D0> with extra args of {}
Traceback (most recent call last):
  File "eventHandler.pyo", line 155, in executeEvent
  File "eventHandler.pyo", line 92, in __init__
  File "eventHandler.pyo", line 100, in next
  File "NVDAObjects\IAccessible\__init__.pyo", line 1478, in event_selection
  File "NVDAObjects\IAccessible\sysListView32.pyo", line 304, in event_stateChange
  File "NVDAObjects\__init__.pyo", line 1016, in event_stateChange
  File "speech.pyo", line 352, in speakObjectProperties
  File "speech.pyo", line 459, in speakText
  File "C:\Users\CB232690\AppData\Roaming\nvda\addons\instantTranslate\globalPlugins\instantTranslate\__init__.py", line 319, in _localSpeak
AttributeError: 'module' object has no attribute 'SPEECH_ITEM_SEPARATOR'
```

## Expected result
No error

## Possible solutions
1. Fix the error
2. Drop the support of NVDA 2019.2.1 that becomes a bit old. If you choose this path, the transition to NVDA 2022.1 could be a good timing to drop Python 2 compatibility.

## Context
I am using NVDA 2019.2.1 for test purpose only.

**Comment by CyrilleB79** [comment]
Hi Beqa
I have installed InstantTranslate 4.4.3, which indicates a compatibility range from 2018.3 to 2022.1.
I confirm that the issue is still present in NVDA2019.2.1.
Could you please fix the errors or update the compatibility range to exclude versions below 2019.3?
Thanks.

**Comment by beqabeqa473** [comment]
Hi.

It should be a trivial fix.
The thing is that i don't remember which python uses nvda 2019.2.
If it is Python2, it will be dropped in the next release version.

**Comment by CyrilleB79** [comment]
NVDA 2019.2 uses Python 2.7. And NVDA 2019.3 uses Python 3.7.

Thus if you do not want to keep and maintain Python 2 code, you should set the first compatible version to 2019.3. IMO that's the best choice.

**Comment by beqabeqa473** [comment]
Yes, I will do that before release.

On 8/31/22, Cyrille Bougot ***@***.***> wrote:
> NVDA 2019.2 uses Python 2.7. And NVDA 2019.3 uses Python 3.7.
>
> Thus if you do not want to keep and maintain Python 2 code, you should set
> the first compatible version to 2019.3. IMO that's the best choice.
>
>
>
> --
> Reply to this email directly or view it on GitHub:
> https://github.com/beqabeqa473/instantTranslate/issues/9#issuecomment-1232214384
> You are receiving this because you commented.
>
> Message ID: ***@***.***>


-- 
with best regards Beqa Gozalishvili
Tell: +995593454005
Email: ***@***.***
Web: https://gozaltech.org
Skype: beqabeqa473
Telegram: https://t.me/gozaltech
facebook: https://facebook.com/gozaltech
twitter: https://twitter.com/beqabeqa473
Instagram: https://instagram.com/beqa.gozalishvili

**Comment by CyrilleB79** [comment]
Fixed in #13.
Closing.
